### DISCUSSION TITLE
Would it be possible to use discord.py v2.0 with this distest?

### DISCUSSION THREAD

**Discussion by NabeelUppel** [JakeCover/distest#82] [Q&A]
I am trying to use threads that is available on v2.0 of discord.py but I don't want to break my old tests that uses distest and discord v.1.7. 

Since distest requires v1 would I be able to use v2.0 of discord.py that I installed using 
`pip install -U git+https://github.com/Rapptz/discord.py`

**Comment by Cobular** [comment]
Maybe? Frankly I don't even know where you're getting discord.py 2.0, I
don't see anything about a 2.0 on their changelogs. Also, this library is
pretty much deprecated now, as I don't have the time to properly maintain
it and the https://github.com/CraftSpider/dpytest library is really just
better. I recommend you check that out!
